src/vmm_mad/remotes/lib/lxd/container.rb

```diff
@@ -177,6 +177,11 @@ def start(options = {})
 
     def stop(options = { :timeout => 120 })
         change_state(__method__, options)
+
```

Comment:
This should be called through the pre/post/clean scripts. This code should be moved to the owsirch network drivers